modules/main-rules/pages/roll-for-close-combat-hits.adoc

```diff
@@ -4,8 +4,9 @@ In close combat Armour values are ignored and instead you get a hit if you equal
 Look up the combat difference on the table to find out the hit numbers for winning and losing units.
 
 Roll a D6 for each of your units in base-to-base contact with the enemy, even if you lost.
-Don't forget that a maximum of two units can attack one enemy unit in close combat.
-You can choose which enemy units to apply hits against but as with shooting you can only put a second hit on a unit once all of the enemy units have been allocated one hit each.
+Don't forget that you can place a maximum of two units in base contact with an enemy unit.
+You can choose which enemy units to apply hits against but as with shooting you can only put a second hit on a unit once all of the enemy units in base contact have been allocated one hit each.
+// IJW editing note: tweaked to use more consistent references to base contact.
```

Comment:
Yup, seems good to me. I was just going to remove the editing note and pass that straight through, and then I spotted some other minor issues in the rest of this bit of text. I suggest his instead.

(Basically I've eliminated passive phrasing, added some punctuation to break up the long sentence a bit, and rephrased because the allocation of shooting hits isn't _quite_ the same because it depends on 'closeness' rather than 'base contact'.) 

```suggestion
Don't forget that you can place a maximum of two units in base contact with an enemy unit.
You can choose which enemy units to apply hits against -- but, similar to the allocation of shooting hits, you may not allocate a second hit to a unit unless you have already allocated a hit to every unit that is in base contact.
// IJW editing note: tweaked to use more consistent references to base contact.
```

Reply:
Maybe take out the editing note too, as it will be obsolete.

Reply:
Oops! Yes.
```suggestion
Don't forget that you can place a maximum of two units in base contact with an enemy unit.
You can choose which enemy units to apply hits against -- but, similar to the allocation of shooting hits, you may not allocate a second hit to a unit unless you have already allocated a hit to every unit that is in base contact.
```